src/v/storage/api.h

```diff
@@ -43,7 +45,10 @@ class api : public ss::peering_sharded_service<api> {
         });
     }
 
+    void stop_waiters() { _has_cluster_uuid_cond.broken(); }
```

Comment:
nit: I'd rename this to `stop_cluster_uuid_waiters`

Reply:
Done